### PR TITLE
Added an extra property assetVersion

### PR DESCRIPTION
**Pull request by sarfarazsheikh10**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -122,4 +129,5 @@ hyperSdkPlugin {
     clientId = getClientId(rootProject)
     sdkVersion = getHyperSDKVersion(rootProject)
     excludedMicroSDKs = getExcludedMicroSdks(rootProject)
+    hyperAssetVersion = getHyperAssetVersion(rootProject)
```
Comment (Subhash703) [inline android/build.gradle:132]: @sarfarazsheikh10 `hyperAssetVersion` does not exist in `hyperSdkPlugin`. Change this to `assetVersion`

Comment (harshgarg18) [reply android/build.gradle:132]: +1

File: android/build.gradle [hunk 2]
```diff
@@ -109,6 +109,13 @@ static def getClientId(rootProject) {
     return rootProject.clientId
 }
 
+static def getHyperAssetVersion(rootProject) {
+    if(!rootProject.hasProperty('assetVersion')) {
```
Comment (harshgarg18) [inline android/build.gradle:113]: call the ext property as hyperAssetVersion

**Review by harshgarg18 — CHANGES_REQUESTED** [review]

File: android/build.gradle (hunk 2 above)
Comment (sarfarazsheikh10) [reply android/build.gradle]: done

File: android/build.gradle [hunk 3]
```diff
@@ -109,6 +109,13 @@ static def getClientId(rootProject) {
     return rootProject.clientId
 }
 
+static def getAssetVersion(rootProject) {
+    if(!rootProject.hasProperty('hyperAssetVersion')) {
+        return ""
```
Comment (Subhash703) [inline android/build.gradle:114]: This won't work. If you send "" build will fail.

Comment (sarfarazsheikh10) [reply android/build.gradle:114]: What should be the default value then?

**Review by JamesGeorg — APPROVED** [review]

**Review by Subhash703 — APPROVED** [review]

**Review by harshgarg18 — APPROVED** [review]

**Pull request merged**